src/components/Toast/Toast.tsx

```diff
@@ -139,17 +136,21 @@ export const toast = ({
                 )}
             </div>
         ),
-        actions ? { important: true, duration: 2500 } : { duration: 2500 } // focused if actionable
+        actions ? { important: true } : {} // focused if actionable
     );
 
-export const Toaster = ({ ...props }) => <SonnerToaster {...props} className="Toaster" />;
-
-Toaster.defaultProps = {
-    // duration: 2500,
-    closeButton: true,
-    offset: 16,
-    gap: 8,
-};
+export const Toaster = ({ duration = 2500, ...props }) => (
```

Comment:
Sorry, only noticed it now. We never really put 2 components in 1 file. Can you move it to a separate one?

Reply:
but in this case it's not a 'component', toast() is a function... should i still seperate them?

Reply:
Clarified f2f. 
`ActionItem` will be separated to its own file/folder.
Main folder and file will be renamed to `Toaster`